bom-internal/pom.xml

```diff
@@ -621,6 +621,11 @@ POM as their parent.
                 <artifactId>commons-lang3</artifactId>
                 <version>3.12.0</version>
             </dependency>
+            <dependency>
```

Comment:
Ah, you're right, we do already have 3.2.2 in the class path, via commons-beanutils. Changed this and my code to use the 3.2.2 version. I didn't create a ticket, since it looks like there are no newer versions of beanutils.